Update PHPUnit configuration

sebastianbergmann opened the pull request.

ondrejmirtes commented:
Hi, thanks for the PR! What are exactly the proposed changes and differences of this configuration? Also, please indent with tabs.

sebastianbergmann commented:
The default values of some configuration settings has changed in PHPUnit 6. This is why `backupGlobals="false"` and `beStrictAboutTestsThatDoNotTestAnything="true"` are no longer required. Removing them has no effect on how the test suite is executed.

The default value of `backupStaticAttributes` is (and was always) `false`. This is why `backupStaticAttributes="false"` is not required. Removing this has no effect on how the test suite is executed.

I added `verbose="true"` to enable more verbose output.

I added the `<testsuite>` element so that the test suite can be executed by simply running `phpunit` in the directory that contains the `phpunit.xml` file.

gsdevme commented:
https://www.youtube.com/watch?v=SsoOG6ZeyUI 😂

ondrejmirtes commented:
Cool, thank you very much!
